frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -23,7 +27,7 @@ const getPVStorageClass = (pv: K8sResourceKind) => _.get(pv, 'spec.storageClassN
 const getStorageClassName = (pvc: K8sResourceKind) =>
   _.get(pvc, ['metadata', 'annotations', 'volume.beta.kubernetes.io/storage-class']) ||
   _.get(pvc, 'spec.storageClassName');
-const isBound = (pvc: K8sResourceKind) => _.get(pvc, 'status.phase') === 'Bound';
+const isBound = (pvc: K8sResourceKind) => _.get(pvc, 'status.phase') === status.BOUND;
```

Comment:
Tests are failing due to this:
```suggestion
const isBound = (pvc: K8sResourceKind) => _.get(pvc, 'status.phase') === status.BOUND);
```

Reply:
This is correct. tests are failing because of this line - https://github.com/openshift/console/pull/5016/files#diff-7879af0cb57fd55121c4d9598ff585ecR69

Reply:
Solved it now.